src/strings/en-us.json

```diff
@@ -1728,5 +1728,20 @@
     "MachineTranslated": "Machine Translated",
     "ForeignPartsOnly": "Forced/Foreign parts only",
     "HearingImpairedShort": "HI/SDH",
-    "LabelIsHearingImpaired": "For hearing impaired (SDH)"
+    "LabelIsHearingImpaired": "For hearing impaired (SDH)",
+    "Creator": "Creator",
+    "Creators": "Creators",
+    "Author": "Author",
+    "Authors": "Authors",
+    "Illustrator": "Illustrator",
+    "Penciller": "Penciller",
+    "Inker": "Inker",
+    "Colorist": "Colorist",
+    "Letterer": "Letterer",
+    "CoverArtist": "Cover artist",
+    "Editors": "Editors",
+    "Editor": "Editor",
+    "Translators": "Translators",
+    "Translator": "Translator",
+    "HeaderCreators": "Creators"
```

Comment:
It would be better to put them in alphabetical order to minimize potential conflicts.

Reply:
Should I sort the entire json file, or juste those new values ?

Reply:
Just new values.